test/e2e/logforwarding/fluentlegacy/forward_to_fluent_test.go

```diff
@@ -84,7 +84,7 @@ var _ = Describe("Backwards compatibility prior to ClusterLogForwarder", func()
 					return e2e.KubeClient.CoreV1().Secrets(fluentDeployment.Namespace).Delete(context.TODO(), secret.ObjectMeta.Name, opts)
 				})
 
-				components := []helpers.LogComponentType{helpers.ComponentTypeCollector, helpers.ComponentTypeStore}
```

Comment:
why removed this?

Reply:
The test is designated to forward logs only to a fluentd receiver and is actually asserting only logs for that output. So why installing an ES at all and churning resources w/o reason?